docs/guide/quickstart.rst

```diff
@@ -0,0 +1,298 @@
+    If you ever leak it, you can click the ``Regenerate`` button to forcibly invalidate all copies of the token. This will also terminate all running instances of your bot.
+
+.. _guide_quickstart_client_secret:
+
+.. warning::
+
+    You may be thinking of using the ``Client Secret`` for your bot token. This is **NOT** what you are looking for - you need the bot **Token**, which is a completely different format.
+
+    .. image:: /images/discord_client_secret_big_nono.png
+        :scale: 90 %
+
+A Simple Bot
+-------------
+
+This step-by-step walk-through will show you how to make a bot using the commands framework.
+
+.. note::
+
+    This walk-through does not cover application commands (slash commands). For a detailed walk-through of application commands, see _guide_app_commands.
+
+
+1. Create a new Python file in the folder you want to work in.
+
+    .. warning::
+
+        Do not name the file ``discord.py``, as this will cause conflicts with the discord.py library. Also, do not create a sub-folder in your project named ``discord``, as that too will cause conflicts.
+
+2. Open the new Python file in your preferred editor.
+
+    If you do not have an editor installed, you can use a community recommended one, such as `Visual Studio Code <https://code.visualstudio.com/>`_,
+    `PyCharm <https://www.jetbrains.com/pycharm/>`_ or `Sublime Text 4 <https://www.sublimetext.com/>`_.
+    We don't recommend Python's built in IDLE, as the lack of features compared to other simple editors makes it very bothersome for projects with many files.
+
+3. Now you can start creating your bot. The following steps will go over a simple bot line-by-line to help you understand what's happening.
+
+First, you need to import the discord.py library:
+
+.. code-block:: python3
+
+    import discord
+    from discord.ext import commands
+
+We import ``discord.ext.commands`` here as we will need it for our bot.
+
+.. code-block:: python3
+
+    intents = discord.Intents.default()
+    intents.message_content = True
+    bot = commands.Bot(command_prefix="!", description="This is my awesome bot!", intents=intents)
+
+This is your bot instance. You can name the variable whatever you like but for simplicity's sake we will name it ``bot``.
+You will need to specify a ``command_prefix`` here, we use ``!``, but you can use any string, or a list of strings for multiple prefixes.
+We also add the :attr:`Intents.message_content` intent, so that our bot can read regular text messages. You can read more in the `Intents guide <_guide_intents>`_.
+
+.. note::
+
+    As of 2.0, the ``intents`` argument is required.
+    ``Intents`` are the method to specify which gateway events you wish to receive. ``Intents.default()`` means you will receive all events that are not locked behind bot verification.
+    For more information, see :ref:`intents_primer`.
```

Comment:
This is unnecessary, the person reading this most likely is only familiar with 2.0. Past facing statements like this are out of place.